src/com/github/TylerWhitmore/cs3230/AssignmentTwo.java

```diff
@@ -0,0 +1,137 @@
+        String input = myScanner.next();
+        int i = 0;
+        try{
+            i = Integer.parseInt(input);
+        }
+        catch (NumberFormatException ex){
+            System.out.println("Invalid response: "+ input);
+        }
+        myScanner.nextLine();
+        return i;
+    }
+
+    private static int[] getNewNumbers(){
+        System.out.println("Please enter a list of integers separated by a space:");
+        String[] userInput = myScanner.nextLine().split("\\s+");
+        int[] integers = new int[userInput.length];
+        for(int i = 0; i < userInput.length; i++){
+            try{
+                integers[i] = Integer.parseInt(userInput[i]);
+            }
+            catch (NumberFormatException ex){
+                System.out.println("Invalid integer: " + userInput[i]);
+            }
+        }
+        return integers;
+    }
+    public static void main(String[] args){
+        boolean cont = true;
+        //clearScreen();
+        //System.out.println("Please enter a list of integers separated by a space:");
+        int[] integers = AssignmentTwo.getNewNumbers();
+        while(cont){
+        System.out.println("Current list of integers:" + Arrays.toString(integers));
+        System.out.println("Choose and operation:");
+        System.out.println("1. Sum - adds all the integers");
+        System.out.println("2. Evens - finds all the evens");
+        System.out.println("3. Odds - finds all the odds");
+        System.out.println("4. Max - finds the largest integer");
+        System.out.println("5. Min - finds the smallest integer");
+        System.out.println("6. Update - enter a new list of integers");
+        System.out.println("7. Exit");
+        int response = AssignmentTwo.getUserInput();
+        switch(response){
+            case 1:
+                System.out.println(">-1");
+                System.out.println(sum(integers));
+                break;
+            case 2:
+                System.out.println(">-2");
+                System.out.println(evens(integers));
+                break;
+            case 3:
+                System.out.println(">-3");
+                System.out.println(odds(integers));
+                break;
+            case 4:
+                System.out.println(">-4");
+                System.out.println(max(integers));
+                break;
+            case 5:
+                System.out.println(">-5");
+                System.out.println(min(integers));
+                break;
+            case 6:
+                System.out.println(">-6");
+                integers = AssignmentTwo.getNewNumbers();
+                break;
+            default:
+                System.out.println(">-7");
+                cont = false;
+                break;
+        }
+    }
+
+    }
+    public static void clearScreen() {  
+        System.out.print("\033[H\033[2J");  
+        System.out.flush();  
+    }
+    public static int sum(int[] arr){
```

Comment:
You’ll need to move these static methods that do the integer logic to their own class. You can name this class whatever you want so long as it makes sense.